simplivity/resources/certificates.py

```diff
@@ -0,0 +1,52 @@
+###
+# (C) Copyright [2020] Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##
+from simplivity.resources.resource import ResourceBase
+
+"""Implements features available for certificate resource."""
+URL = '/security/certificates'
+DATA_FIELD = 'certificates'
+
+
+class Certificates(ResourceBase):
+    """Implements features for SimpliVity Certificates resources."""
+
+    def __init__(self, connection):
+        """Initialize Certificates class."""
+        super(Certificates, self).__init__(connection)
+
+    def get_all(self):
+        """Get all SSL certificates from the HPE SimpliVity trust store"""
+        return self._client.get_all(URL, members_field=DATA_FIELD)
+
+    def get_by_data(self, data):
```

Comment:
Is this resource supporting get_by_id and get_by_name methods? if yes add an example for those methods in the example file, if not let's override those methods here and return a method not available exception.

Reply:
Added new exception Method not found and return the same from the get_by_id and get_by_name method